libs/community/langchain_community/document_loaders/blob_loaders/__init__.py

```diff
@@ -4,6 +4,9 @@
 from langchain_core.document_loaders import Blob, BlobLoader
 
 if TYPE_CHECKING:
+    from langchain_community.document_loaders.blob_loaders.cloud_blob_loader import (
+        CloudBlobLoader,
+    )
     from langchain_community.document_loaders.blob_loaders.file_system import (
         FileSystemBlobLoader,
```

Comment:
**Static Code Issue**: Move import 'langchain_community.document_loaders.blob_loaders.file_system.FileSystemBlobLoader' out of type-checking block. Import is used for more than type hinting. <br> **Fix**: Move the import statement for 'FileSystemBlobLoader' outside the TYPE_CHECKING block to the top-level imports. <br> **Code Suggestion**: 
 ```
 4a6
 > from langchain_community.document_loaders.blob_loaders.file_system import FileSystemBlobLoader
 11,13d12
 <     from langchain_community.document_loaders.blob_loaders.file_system import (
 <         FileSystemBlobLoader,
 <     )
 ```